src/OrchardCore.Modules/OrchardCore.Resources/ResourceManifest.cs

```diff
@@ -217,7 +217,7 @@ public void BuildManifests(IResourceManifestBuilder builder)
 
             manifest
                 .DefineScript("resizable-resolveconflict")
-                .SetDependencies("jQuery")
+                .SetDependencies("jQuery-ui")
```

Comment:
does resizable require that?

Reply:
Yes, the resizable-resolveconflict is a script that fix the name conflict between the resizable plugin versione of jquery ui and the resizable plugin version used by trumbowyg. Is necessary that it run after jQuery ui is loaded.